tests/test_download.py

```diff
@@ -63,3 +66,89 @@ def test_download(tmpdir, monkeypatch, requests_mock):
     get_input_vfs("example")
 
     assert requests_mock.call_count == 3
+
+
+@pytest.mark.parametrize(
+    "datum_name",
+    [
+        "01941935-832f-16d4-af69-6a9bf6bf4df6",
+        "my_models_alias",
+    ],
+)
+def test_datum_url_download(tmpdir, monkeypatch, requests_mock, datum_name):
+    inputs_dir = str(tmpdir.mkdir("inputs"))
+    project_dir = str(tmpdir.mkdir("project"))
+    monkeypatch.setenv("VH_INPUTS_DIR", inputs_dir)
+
+    project_id = "018a5fc7-02f8-d42d-a22b-cb6ffe20919e"
+    datum_id = "01941935-832f-16d4-af69-6a9bf6bf4df6"
+    datum_alias = "my_models_alias"
+    vh_host = "https://app.valohai.com"
+
+    # Final file to download
+    requests_mock.get(
+        "https://valohai-mnist.s3.amazonaws.com/t10k-images-idx3-ubyte.gz",
```

Comment:
Same with the file name
as and use the variable where it's required
`filename = t10k-images-idx3-ubyte.gz `